eth-connector/src/operation.rs

```diff
@@ -1,5 +1,6 @@
-use crate::types::{MigrationCheckResult, PausedMask, WithdrawResult};
-use aurora_workspace_types::AccountId;
+use crate::types::{MigrationCheckResult, PausedMask};
+use aurora_engine_types::account_id::AccountId;
+use aurora_engine_types::parameters::connector::WithdrawResult;
```

Comment:
Unfortunately, we can do that. `aurora-workspace` is used in `aurora-eth-connecotr`. And it has its own `WithdrawResult`, and it can't depend on `aurora-engine-types` in that particular case. 

It was nothing bad to have that type here, because we can use it easily for tests in `Engine` and `aurora-et-connector`.